moment_kinetics/src/time_advance.jl

```diff
@@ -475,13 +478,15 @@ function setup_advance_flags(moments, composition, t_input, collisions,
     advance_neutral_energy = false
     r_diffusion = false
     vpa_diffusion = false
+    vperp_diffusion = false
     vz_diffusion = false
     explicit_weakform_fp_collisions = false
     # all advance flags remain false if using operator-splitting
     # otherwise, check to see if the flags need to be set to true
     if !t_input.split_operators
         # default for non-split operators is to include both vpa and z advection together
         advance_vpa_advection = true && vpa.n > 1 && z.n > 1
+        advance_vperp_advection = true && vperp.n > 1 && z.n > 1
         advance_z_advection = true && z.n > 1
         advance_r_advection = true && r.n > 1
```

Comment:
Not introduced in this PR, bu the `true &&` looks strange and does not do anything. I think that is just confusing, so we might as well remove.
```suggestion
        advance_vpa_advection = vpa.n > 1 && z.n > 1
        advance_vperp_advection = vperp.n > 1 && z.n > 1
        advance_z_advection = z.n > 1
        advance_r_advection = r.n > 1
```